### PR TITLE
getMenteesForMentor endpoint (control, model, routes), awaiting Mongo

### PR DESCRIPTION
**Pull request by txingxie**

### CODE REVIEW

File: api/src/server.ts [hunk 1]
```diff
@@ -19,6 +20,8 @@ app.use("/workshop", routes.workshop);
 
 connectDB();
 
+app.use("/api", router); // connect routes (prefix as api), what is the lin 7 logic for?: import * as routes from "./routes/index"
```
Comment (Lyton505) [inline api/src/server.ts:23]: It imports modules user and workshop on [line 18 in this file](https://github.com/ChangePlusPlusVandy/PennsylvaniaWomenWork/blob/7c66c90a8230535c54a8b520ef6f40a4410add15/api/src/routes/index.ts#L18)

**Comment by txingxie** [comment]
Tested w/ Mongo Implementation & Postman:

**This request:** GET - http://localhost:8000/api/mentor/674d179b846a2c3069b9cfbb/mentees
**Returns this:**
[
    {
        "_id": "674d167761974e2844f17a07",
        "firstName": "MenteeTestFirstName",
        "lastName": "MenteeTestLastName",
        "username": "MenteeTestUsername",
        "email": "menteetestemail@gmail.com",
        "role": "mentee",
        "workshopIDs": [],
        "menteeInfo": [],
        "meetingSchedule": [],
        "meetings": [],
        "__v": 2
    },
    {
        "_id": "674d16a961974e2844f17a09",
        "firstName": "MenteeTest2FirstName",
        "lastName": "MenteeTest2LastName",
        "username": "MenteeTest2Username",
        "email": "menteetest2email@gmail.com",
        "role": "mentee",
        "workshopIDs": [],
        "menteeInfo": [],
        "meetingSchedule": [],
        "meetings": [],
        "__v": 2
    }
]

**Added the following objects to Mongo:**
_Mentor_
{"_id":{"$oid":"674d179b846a2c3069b9cfbb"},"firstName":"MentorTestFirstName","lastName":"MentorTestLastName","username":"MentorTestUsername","email":"mentortest@gmail.com","role":"mentor","workshopIDs":["workshopID1"],"menteeInfo":[{"$oid":"674d167761974e2844f17a07"},{"$oid":"674d16a961974e2844f17a09"}],"meetingSchedule":[],"__v":{"$numberInt":"0"}}

_Mentee 1_
{"_id":{"$oid":"674d167761974e2844f17a07"},"firstName":"MenteeTestFirstName","lastName":"MenteeTestLastName","username":"MenteeTestUsername","email":"menteetestemail@gmail.com","role":"mentee","workshopIDs":[],"menteeInfo":[],"meetingSchedule":[],"meetings":[],"__v":{"$numberInt":"2"}}

_Mentee 2_
{"_id":{"$oid":"674d16a961974e2844f17a09"},"firstName":"MenteeTest2FirstName","lastName":"MenteeTest2LastName","username":"MenteeTest2Username","email":"menteetest2email@gmail.com","role":"mentee","workshopIDs":[],"menteeInfo":[],"meetingSchedule":[],"meetings":[],"__v":{"$numberInt":"2"}}

**MUST ENSURE THAT MENTEES IN THE 'menteeInfo' FIELD ARRAY ARE REPRESENTED BY ObjectIDs RATHER THAN STRINGS.**

**Review by lolitaroz — APPROVED** [review]
looks good to me -- resolve merge conflicts and it can be merged!

**Comment by txingxie** [comment]
I resolved conflicts in the web editor but could someone take a second look at api/src/server.ts, specifically the app.use statements? previously there was a "app.use("/api", routes.workshop)" which might interfere with "app.use("/api", router)". Instead I added "app.use("/workshop", routes.workshop)" and kept the /user and /api statements, for which the backend worked on my local repo

**Review by lolitaroz — APPROVED** [review]
i think it should work! you can merge

**Pull request merged**